### PR TITLE
OAP-53 Fix engine Dockerfile, build psycopg2 from source not binary, write daemon

### PR DESCRIPTION
**Pull request by celinanperalta**
Adds `libpq5` to build and replaces `psycopg2-binary` with `psycopg2`, the source package, which is recommended for production. Error that I encountered is solved by this: https://stackoverflow.com/questions/65352104/how-to-solve-libpq-so-5-import-error-when-running-postgresql-on-alpine-in-docker

Adds daemon code: Runs initial clean and seed on every startup, runs periodic refresh (currently set to 50 items every 5 minutes for testing)

### CODE REVIEW

**Review by eshellman — COMMENTED** [review]
nice to know about libpq5. 
Is there some sort of versioning needed to prevent future breakage?

**Comment by celinanperalta** [comment]
> nice to know about libpq5. Is there some sort of versioning needed to prevent future breakage?

I think the only version that needs to be pinned is for `psycopg2` version 2.9.3. From what I've read, I don't believe that I need to specify a specific version of `libpq5`.

File: oapen-engine/src/tasks/daemon.py [hunk 1]
```diff
@@ -12,10 +18,30 @@ def signal_handler(signal, frame):
 
 
 print("Daemon up and running")
+print("STARTED: clean.py")
+run_clean()
```
Comment (celinanperalta) [inline oapen-engine/src/tasks/daemon.py:22]: Should we be running a clean on every daemon startup?

Comment (justinoboyle) [reply oapen-engine/src/tasks/daemon.py:22]: Probably not, can maybe have an env variable that flushes it on startup for dev?

**Review by justinoboyle — APPROVED** [review]

File: oapen-engine/src/tasks/seed.py [hunk 2]
```diff
@@ -95,9 +96,7 @@ def main():
 
     db_pool = concurrent.futures.ThreadPoolExecutor()
     io_pool = concurrent.futures.ThreadPoolExecutor(max_workers=config.IO_MAX_WORKERS)
-    ngrams_pool = concurrent.futures.ProcessPoolExecutor(
-        max_workers=config.NGRAMS_MAX_WORKERS
-    )
+    ngrams_pool = concurrent.futures.ProcessPoolExecutor(max_workers=cpu_count())
```
Comment (justinoboyle) [inline oapen-engine/src/tasks/seed.py:99]: Nice good call

**Pull request merged**